.github/ISSUE_TEMPLATE/1-bug_report.yml

```diff
@@ -0,0 +1,64 @@
+name: Bug Report
+description: Template for reporting bugs
+title: "[BUG]: "
+labels: "bug"
+body:
+  - type: markdown
+    attributes:
+      value: Tech support does not belong here. You should only file an issue here if you think you have experienced an actual bug with Vita3K.
+             We are **not** accepting bug reports for specific games at this time.
```

Comment:
I think this line should be removed. It's not that we are not accepting game specific bug reports but indeed as you say below that they must go to the game specific issue on the compatibility repository.

Reply:
If you say so.